### PR TITLE
Monosetting enum

### LINKED ISSUE #380: Set wavelength when calibration is not specified
The wavelength should be set on the `HidraWorkspace` during reduction even when no calibration is specified. The NeXus file specifies it indirectly through `mrot`.

It is currently in [calibration](https://github.com/neutrons/PyRS/blob/master/pyrs/calibration/peakfit_calibration.py#L101-L119) and should be converted to a standalone `Enum`. It would work best if `__float__` returned the wavelength, and `__str__` returned the name.

### PR DESCRIPTION
**Pull request by quantumsteve**
This adds the Monosetting Enum to obtain an approximate wavelength from the monochromator rotation angle

Fixes #380

### CODE REVIEW

File: pyrs/calibration/peakfit_calibration.py [hunk 1]
```diff
@@ -77,6 +78,51 @@ def __init__(self):
...
+        elif -170.0 < mrot < -166.0:
+            monosetting = MonoSetting('Si331')
+        elif 14.0 < mrot < 18.0:
+            monosetting = MonoSetting('Si400')
+        elif -11.0 < mrot < -8.0:
+            monosetting = MonoSetting('Si311')
+        elif -185.0 < mrot < -180.0:
+            monosetting = MonoSetting('Si220')
+        else:
+            raise ValueError('Unable to determine monosetting from the monochromator rotation angle')
```
Comment (peterfpeterson) [inline pyrs/calibration/peakfit_calibration.py]: Add the value to the exception message so people know what specifically didn't work

File: pyrs/calibration/peakfit_calibration.py [hunk 2]
```diff
@@ -77,6 +78,51 @@ def __init__(self):
...
+    def __str__(self):
+        return self.value
+
+    def __float__(self):
+        return self.wavelength
+
+    @staticmethod
+    def getFromRotation(mrot):
+        if -41.0 < mrot < -38.0:
+            monosetting = Monosetting('Si333')
```
Comment (peterfpeterson) [inline pyrs/calibration/peakfit_calibration.py]: All of these should call the enum values directly
```suggestion
            monosetting = Monosetting.Si333
```

File: pyrs/calibration/peakfit_calibration.py [hunk 3]
```diff
@@ -126,8 +157,7 @@ def __init__(self, hb2b_instrument, hidra_data=None, hidra_data2=None, scheme=1)
         self.tth_ref = '2thetaSetpoint'
 
         # Set wave length
-        self._calib[6] = \
-            np.array([1.452, 1.452, 1.540, 1.731, 1.886, 2.275, 2.667])[monosetting]
+        self._calib[6] = self.monosetting.wavelength
```
Comment (quantumsteve) [inline pyrs/calibration/peakfit_calibration.py]: @fanchercm Does Si333 and Si511 have the same wavelength?

File: pyrs/calibration/peakfit_calibration.py (hunk 2 above)
Comment (quantumsteve) [reply pyrs/calibration/peakfit_calibration.py]: done

File: pyrs/calibration/peakfit_calibration.py (hunk 1 above)
Comment (quantumsteve) [reply pyrs/calibration/peakfit_calibration.py]: done

**Pull request merged**